lib/developer_portal/app/controllers/developer_portal/base_controller.rb

```diff
@@ -25,4 +25,11 @@ def finish_signup_for_paid_plan
 
     redirect_to edit_payment_details_path
   end
+
+  def filter_readonly_params(params, resource_class)
+    return unless params
+
+    read_only_fields = FieldsDefinition.by_provider(site_account).by_target(resource_class.name).select(&:read_only).map(&:name)
```

Comment:
`.read_only.pluck(:name)` if we add the scope I mentioned above.

Reply:
so I guess now just 
```suggestion
    read_only_fields = FieldsDefinition.by_provider(site_account).by_target(resource_class.name).read_only.pluck(:name)
```

Reply:
Right: https://github.com/3scale/porta/pull/3997/commits/be1f2050bf4f791cb35cc9acb6674ab67b51676d